backends/bmv2/jsonconverter.cpp

```diff
@@ -836,6 +885,9 @@ JsonConverter::convertActionBody(const IR::Vector<IR::StatOrDecl>* body,
                                  Util::JsonArray* result, Util::JsonArray* fieldLists,
                                  Util::JsonArray* calculations, Util::JsonArray* learn_lists) {
     for (auto s : *body) {
+        // TBD jafinger - add line/col at all individual cases below,
```

Comment:
so far we have TODO, FIXME, and a few XXX.
Perhaps we should use TODO instead of TBD: makes it simpler to grep.